### PR TITLE
[master] add models

### PR DESCRIPTION
**Pull request by plokkkk**
add Heritage models
Module to import tsv to db

### CODE REVIEW

**Comment by FranckKe** [comment]
What is the command to import the data ? Would be nice to include it in the readme.

**Comment by plokkkk** [comment]
Update readme with the command to import data to database and which columns we use.

File: imports/cli.go [hunk 1]
```diff
@@ -70,8 +70,7 @@ func action(context *cli.Context) error {
 			h.LieuDit = record[header["lieu_dit"]]
 			h.Datings = models.NewStringSlice(record[header["datations_principales"]], record[header["datations_secondaires"]])
 			h.Status = record[header["statut"]]
-			h.Study = record[header["cadre_etude"]]
-			h.Study = record[header["cadre_etude"]]
+			h.Study = record[header["cadre_etude"]
```
Comment (mdouchement) [inline imports/cli.go]: - missing closing bracket!

File: imports/cli.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+		} else {
+			h := models.NewHeritage()
+			h.Longitude, _ = strconv.ParseFloat(record[header["longitude"]], 64) // ignore error
+			h.Latitude, _ = strconv.ParseFloat(record[header["latitude"]], 64)   // ignore error
+			h.Address = record[header["adresse"]]
+			h.Commune = record[header["commune"]]
+			h.LieuDit = record[header["lieu_dit"]]
+			h.Datings = models.NewStringSlice(record[header["datations_principales"]], record[header["datations_secondaires"]])
+			h.Status = record[header["statut"]]
+			h.Study = record[header["cadre_etude"]
```
Comment (mdouchement) [inline imports/cli.go]: - missing closing bracket!

Comment (plokkkk) [reply imports/cli.go]: oops

Comment (mdouchement) [reply imports/cli.go]: - To avoid this, just run the server to run the Go static analysis during build.

File: README.md [hunk 3]
```diff
@@ -30,6 +30,28 @@ $ find . -name '*.go' -not -path './vendor*' -exec go fmt {} \;
 
 > Environment variables https://github.com/mdouchement/breizhtrip-go/blob/master/config/config.go
 
+## Import data to database
+
+You need to have a database name "breizhtrip_development".
```
Comment (mdouchement) [inline README.md]: - or `breizhtrip_production` or `breizhtrip_test` according to the Gin Gonic mode.

File: README.md [hunk 4]
```diff
@@ -32,7 +32,11 @@ $ find . -name '*.go' -not -path './vendor*' -exec go fmt {} \;
 
 ## Import data to database
 
-You need to have a database name "breizhtrip_development".
+You need to have a database name:
+* "breizhtrip_development" or
+* "reizhtrip_production" or
+* "breizhtrip_test"
+
```
Comment (FranckKe) [inline README.md]: Suggest edit:
You need a database named "breizhtrip_" + `environment name`. It can be `development`, `test` or `production`

Comment (mdouchement) [reply README.md]: - Nope, Gin does not use these environment names

File: README.md [hunk 5]
```diff
@@ -30,6 +30,32 @@ $ find . -name '*.go' -not -path './vendor*' -exec go fmt {} \;
 
 > Environment variables https://github.com/mdouchement/breizhtrip-go/blob/master/config/config.go
 
+## Import data to database
+
+You need to have a database name:
+* "breizhtrip_development" or
+* "reizhtrip_production" or
```
Comment (mdouchement) [inline README.md]: - typo

File: README.md (hunk 4 above)
Comment (mdouchement) [reply README.md]: Auto create database is added to this PR

**Pull request merged**